atoMEC/__init__.py

```diff
@@ -179,6 +169,7 @@ def radius(self, radius):
 
     @property
     def density(self):
+        """float: the mass density of the material in units :math:`\mathrm{g\ cm}^{-3}`"""
```

Comment:
Unit does not get rendered correctly. Turning string into a raw string fixes it.
```suggestion
        r"""float: the mass density of the material in units :math:`\mathrm{g\ cm}^{-3}`"""
```